test_sets/PTU_no_certificate_exists_at_module_config.txt

```diff
@@ -0,0 +1,57 @@
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/012_ATF_P_TC_default_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/013_ATF_P_TC_media_app_PTU_retry_sequence_finish_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/014_ATF_P_TC_comm_app_PTU_retry_sequence_finish_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/015_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/016_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_ON_StartService_encrypted_true.lua Check_that_SDL_will_return_StartService_Audio_Video_RPC_Hybrid_NACK_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/017_ATF_P_TC_media_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/018_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/019_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/020_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/021_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_Audio_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/022_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_Video_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/023_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_Hybrid_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
```

Comment:
@OVikhrov, according to clarification "Can you clarify how is expected sequence to acknowledge Services?"
Hybrid service should be the same of RPC service, only ServiceType: 15 + binaryData

Regarding secure service - yes, it's set encryption flag to true.